assets/js/translator.js

```diff
@@ -188,6 +217,90 @@ if(modeEnabled('translation')) {
             $('a#fileDownload').fadeOut('fast');
         });
 
+        $('#translatedText').css('height', $('#originalText').css('height'));
+        $('#suggestCloseBtn').click(function() {
+            $('#suggestedWordInput').val('');
+            grecaptcha.reset();
+        });
+        $('#suggestBtn').click(function() {
+            var fromWord = $('#suggestionTargetWord').html();
+            var toWord = $('#suggestedWordInput').val();
+            var recaptchaResponse = grecaptcha.getResponse();
+
+            if(toWord.length === 0) {
+                $('#suggestedWordInput').tooltip('destroy');
+                $('#suggestedWordInput').tooltip({
+                    'title': 'Suggestion cannot be empty.',
+                    'trigger': 'manual',
+                    'placement': 'bottom'
+                });
+                $('#suggestedWordInput').tooltip('show')
+                setTimeout(function() {
+                    $('#suggestedWordInput').tooltip('hide');
```

Comment:
Do we really need to hide it before destroying or does this actually do something? o.o